gprproject/gprbuild.py

```diff
@@ -277,6 +288,7 @@ def save(self):
             "symcc": self.symcc,
             "prefix": self.prefix,
             "gpr_paths": self.gpr_paths,
+            "gpr_opts": self.gpr_opts,
```

Comment:
In response to https://github.com/AdaCore/gnatcoll-core/pull/94/files#r1881804015:
To make it work properly, this line needs to be removed. Each time the build command is run, its environment and provided arguments are saved in a JSON file for reuse by the clean, uninstall, and install commands. You can search for "load" in `gprproject/init.py` for reference. If we store `gpr_opts` arguments here, they may not be valid for the other tools. For instance:

```
$ gnatcoll_core.gpr.py build --gpr-opts="-O0"
$ gnatcoll_core.gpr.py install <<---- raises gprinstall: unrecognized option '-O0'
```

Reply:
This is a good solution! I'll review your new PR if you plan to create one.